packages/admin/dashboard/src/routes/orders/order-create-exchange/components/exchange-create-form/exchange-create-form.tsx

```diff
@@ -498,6 +522,50 @@ export const ExchangeCreateForm = ({
                 </span>
               </div>
             </div>
+
+            {/* CARRY OVER PROMOTION */}
+            {hasPromotions && (
+              <div className="bg-ui-bg-field mt-4 rounded-lg border py-2 pl-2 pr-4">
+                <Form.Field
+                  control={form.control}
+                  name="carry_over_promotion"
+                  render={({ field: { onChange, value, ...field } }) => {
+                    return (
+                      <Form.Item>
+                        <div className="flex items-center">
+                          <Form.Control className="mr-4 self-start">
+                            <Switch
+                              dir="ltr"
+                              className="mt-[2px] rtl:rotate-180"
+                              checked={!!value}
+                              onCheckedChange={async (checked) => {
+                                onChange(checked)
+                                if (preview?.order_change?.id) {
+                                  await updateOrderChange({
+                                    carry_over_promotions: checked,
+                                  })
+                                }
+                              }}
+                              {...field}
+                            />
+                          </Form.Control>
+                          <div className="block">
+                            <Form.Label>
+                              {t("orders.exchanges.carryOverPromotion")}
+                            </Form.Label>
+                            <Form.Hint className="!mt-1">
+                              {t("orders.exchanges.carryOverPromotionHint")}
```

Comment:
nit: can we add a tooltip that also explains what types of promotions this is applicable to?